docs/specification/README.md

```diff
@@ -275,6 +275,8 @@ A Statistics record contains summary information about the recorded data. The st
 | 4 | attachment_count | uint32 | Number of Attachment records in the file. |
 | 4 | metadata_count | uint32 | Number of Metadata records in the file. |
 | 4 | chunk_count | uint32 | Number of Chunk records in the file. |
+| 8 | start_time | Timestamp | Earliest message log_time in the file. |
+| 8 | end_time | Timestamp | Latest message log_time in the file. |
```

Comment:
should we call them `message_start_time`/`message_end_time`? or `min_message_time`/`max_message_time`?